### ISSUE TITLE
List doesn't load

### ISSUE THREAD

**Issue by traniix** [kodi-pvr/pvr.iptvsimple#673]
- Hey I have an issue for about 2 months the iptv client doesn't load the channels. On any other iptv client it works. It worked before but for 2 months it doesn't work I don't know if an update causes this problem because on older client versions with older kodi versions it still works.

**Comment by phunkyfish** [comment]
Do you have a full debug log? Otherwise there is no way for me to tell what’s happening.

https://kodi.wiki/view/Log_file/Easy

**Comment by traniix** [comment]
[kodi.log](https://github.com/kodi-pvr/pvr.iptvsimple/files/10253046/kodi.log)

**Comment by traniix** [comment]
Hello I wanted to ask if u looked further into my problem

**Comment by phunkyfish** [comment]
I see this error in the log:

`2022-12-18 03:09:37.990 T:756     ERROR <general>: AddOnLog: pvr.iptvsimple: pvr.iptvsimple - iptvsimple::PlaylistLoader::LoadPlayList - Unable to load playlist cache file 'http://fastrunner.live:8080/get.php?username=USER&password=PASS&type=m3u_plus&output=m3u8':  file is missing or empty.`

Maybe your provider does not support the kodi user agent or something.

**Comment by phunkyfish** [comment]
Can you try manually downloading the file to your computer and load it locally?

**Comment by traniix** [comment]
Yes I tried it with the local file and the channels load but nothing plays. I get an error message. I don't think my provider is the problem because it was working before and now I doesn't. And it works on older client versions.

**Comment by phunkyfish** [comment]
Please post a full debug log of trying to play the local M3U file. But my guess is the error will be the same, but let’s see.

**Comment by basilgello** [comment]
Also select category logging and tick libcurl in the category list box. It will show cURL requests and responses. Alternatively, try spoofing User-Agent to Mozilla: `http://url|User-Agent=Mozilla` and see if the stream becomes playable.
